### DISCUSSION TITLE
payment declined

### DISCUSSION THREAD

**Discussion by jbunds** [community/community#64103] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I need to upgrade to a paid subscription to increase my GitHub Actions resource quota.

I am trying to do that using the standard form [here](https://github.com/settings/billing/payment_information).

My VISA credit card has been repeatedly declined, and I cannot determine why.

I called the bank who issued my VISA credit card and verified that (1) there is plenty of balance available, and (2) that they have not received any payment authorization requests for my VISA credit card, after github.com has reported the following message multiple times:

```
The credit card was declined by the payment processor, please verify the card's information.
If the problem persists, please contact the card's issuer.
```

Well, I **_have_** contacted my VISA credit card's issuer, and they have verified that the problem is not on my end.

So I need to speak with someone responsible for user billing at GitHub, but according to [official GitHub user support documentation](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket), that avenue appears to be unavailable to me as I am not yet a paying GitHub customer.

If anyone who stumbles upon this help request knows the person or team at GitHub (or perhaps the apparent [zuora.com](https://zuora.com) payment intermediary) who can assist me, I would be grateful if you could show this post to them.

**Comment by christophercouch** [comment, marked as answer]
I'm sorry for the inconvenience you're facing. Please try the following:

1. Double-check the card details you've entered, especially the expiration date and CVV.
2. Ensure your billing address matches the one on your credit card.
3. Try using a different browser or clearing your browser cache.
4. Consider trying a different credit card if available.

If all fails, you can reach out to GitHub Support via [this form](https://support.github.com/contact) even if you're not a paying customer yet. Provide them with details of the issue, and they should be able to assist further.

**Reply by ottokovarik** [reply]
I had the same issue and step 2 helped on my side: My name has diacritics (ˇ') but on the card the name is without, so the prefilled credentials in billing info can't be with diacritics - I can imagine this can happen to many people with "non-english" name...

**Reply by vulbsti** [reply]
For me Step 3 worked out. Turns of using different browser was the solution. Because I have too many extensions installed on the primary one that was blocking the card.

**Reply by Shougriver** [reply]
Tried all these to no avail. Multiple cards on multiple browsers on multiple machines all fail. Had to ultimately use the app store to subscribe, but then i can only do month to month

**Reply by thpnt** [reply]
If not resolved. Download Github on App Store and upgrade your copilot with Apple Pay.

**Reply by thsilvadev** [reply]
For some odd reason, in my billing information my country was swapped to another one :x

Switching it back to my true country and trying to add the payment method again worked!

**Reply by Jazzbea** [reply]
This is not helpful. Do I have to take out new CC's just to continue supporting someone's work?

**Reply by vfiruz97** [reply]
For me the 3rd Step was helpful.

**Comment by sushyl** [comment]
I had the same issue and using different browser helped me.

**Reply by samus-aran** [reply]
Welcome to the GitHub Community, @sushyl , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by suleman-the-stammer** [reply]
In your billing address section give address that is same of your credit card address

**Reply by Alegruz** [reply]
Naver Whale definitely doesn't work. Using Edge fixed the bug.

**Reply by pointmina** [reply]
> Naver Whale definitely doesn't work. Using Edge fixed the bug.

This worked for me! Thanks for the tip.

**Comment by uriinf** [comment]
Try to disable browser extensions which blocks third-party trackers. I have Privacy Badger extension, which was blocking *.stripe.com on the Payment method update page. Once I've disabled the extension - card is added without any issues.

**Comment by dev-vikassaini** [comment]
Hi, I am also getting the same issue even though I use a couple of cards

**Comment by markgnoble** [comment]
yeah I had this - I was using a VPN with add-block etc. disabled VPN made transaction first time

**Comment by Coderxrohan** [comment]
Hi @dev-vikassaini,

I faced a similar issue before. Here are a few things you can try that helped others:

Double-check that your billing address matches exactly with what’s on file with your card issuer.

Try using a different browser or clear your cache.

Disable any browser extensions that block trackers or ads (e.g., Privacy Badger, uBlock, etc.).

If you’re using a VPN, turn it off and try again.

Make sure your name is entered exactly as it appears on the card (without any special characters or accents if your card doesn’t have them).

If possible, try another card.

If none of this works, you can still contact GitHub Support directly using [this form](https://support.github.com/contact). Even if you don’t have a paid plan yet, they can help you sort out billing issues.

Hope this helps!

**Reply by Shougriver** [reply]
I am having this exact same issue. I have tried multiple browsers from multiple different devices with multiple different credit cards. I have tried over cellular and over wifi... I have tried all the suggestions in your post to no avail.

Ultimately had to use the Apple app store to subscribe, but the app store only allows month to month billing at the current rates and will not allow me to select the 1-year plan for the discounted rate.

I have raised a ticket with GitHub but get the typical "we're too busy to respond rn" message

**Comment by Ridermansb** [comment]
This is ridiculous. I use the same card for all other services, but it is not working here. GitHub is refusing to accept payments.

I tried several other cards, but the same issue occurred!

I attempted the following:
1. Different browsers (for instance, Safari)
2. Different cards (but encountered the same issue)

**Reply by biswajitkarlumen** [reply]
Are you able to do? I am stuck and raised a ticket also, but no has replied since 3 days.

**Comment by ayla747108** [comment]
That sounds really frustrating! Since you’ve confirmed the issue isn’t with your card, the problem might be on GitHub’s payment processor side (Zuora). I’d suggest reaching out to GitHub Support via their contact form for account issues
 and clearly mentioning that your card is valid but keeps being declined. Sometimes tagging them on Twitter (@GitHubHelp) can also get a faster response. Hopefully, they can get your subscription upgraded soon!

**Reply by Shougriver** [reply]
Thanks. I submitted a ticket last week. Got the typical "high volume" canned email response. We'll see when they get back to me I guess...

**Comment by Sikaerus** [comment]
It's ridiculous how, even today, the problem persists... We don't need solutions, you need solve it. 
Two years... How many AIs and time do you need to fix this issue?

**Reply by jeckyl2010** [reply]
I also see this today. Tried 3 cards from 3 different browsers and 2 different computers :(

**Comment by jeckyl2010** [comment]
Hi Surya

Thanks for the quick reply. I wonder if you actually ready my ticket, because I did write that I have tried numerous times, double checked the details, used multiple card, from multiple browsers and multiple computers - nothing worked, and my card issuer can't see anything in the logs.

/Anders
________________________________

**Comment by DaboinCJ** [comment]
In my case, the issue was number 2. I recommend opening your credit card statement and using your name and address exactly as displayed. In my case the credit card statement had incomplete versions of my name and address. The card was only saved once I inserted those incomplete text samples into the payment engine.

**Comment by sqrtv2** [comment]
same problem - The credit card was declined by the payment processor, please verify the card's information. If the problem persists, please contact the card's issuer.
wtf?

**Comment by dromzeh** [comment]
Had the same issue, tried multiple cards, all with the same err. Checking the network tab revealed 'The request is temporarily blocked due to security reason.' 

Fixed by using another device.

**Comment by Mehrab-Javadzadeh** [comment]
Hi everyone,

I'm having a very frustrating billing issue. I'm trying to upgrade to a paid subscription to increase my Actions quota, but my VISA card is being repeatedly declined. My bank has confirmed there are sufficient funds and that they haven't even received a request from GitHub. This seems to be a problem with GitHub's payment processor.

The good news is that this is a common problem, and the community has found several solutions that don't require you to be a paying customer to get help.

Here are the steps that have worked for others:

1.  **The "Easy Fix" Checklist:** Before anything else, double-check these details [citation:2]:
    *   Your billing address matches your card's address exactly [citation:2].
    *   Your name is entered precisely as it appears on the card (pay attention to accents or special characters) [citation:2].
    *   Try a different browser or clear your cache and cookies [citation:2].
    *   Disable all browser extensions, especially ad-blockers, privacy tools (like Privacy Badger), or VPNs, which can interfere with the payment form [citation:2][citation:10].

2.  **Contact GitHub Support:** This is the most important step. **You can contact GitHub billing support even if you are not a paying customer** [citation:2][citation:5][citation:9].
    *   Go to the GitHub Support contact page: **https://support.github.com/contact/account** [citation:5][citation:9].
    *   Select **"Billing"** or **"Billing & Payments"** as the issue type [citation:2][citation:5][citation:9].
    *   Explain the situation clearly: include your username, the fact that you're trying to upgrade, that your bank sees no declined charges, and list the troubleshooting steps you've already taken. Mention you suspect the issue is with the payment processor (Zuora or Stripe) [citation:2][citation:10].

3.  **Alternative Solutions:** If contacting support is not an option or is taking too long, some users have had success with [citation:4][citation:12]:
    *   Trying a completely different card (some found a virtual credit card worked when their physical one didn't) [citation:4].
    *   Switching from PayPal to a credit card (or vice-versa) [citation:12].
    *   If you have a "budget" set, temporarily increasing it to a value like $1 to "unlock" the payment flow [citation:4].

I hope this helps us both get this sorted out!

**Comment by FergusRooney** [comment]
I had this issue. I managed to get it working by adding the card to my billing account first
Try going to account settings -> Billing and licensing -> Payment information -> Payment method
And add your card there instead
